source/ways-of-working/environments/external-access.html.md.erb

```diff
@@ -0,0 +1,81 @@
+---
+title: External Access in Demo
+last_reviewed_on: 2022-03-30
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+The demo environment is used to demonstrate new features to external stakeholders. 
+
+For guidance on how to deploy an application to the demo environment, click [here](https://hmcts.github.io/ways-of-working/common-pipeline/common-pipeline.html#building-infrastructure-in-the-demo-environment).
+
+There are three different ways in which an application in demo can be made available to external users. This is dependent on the ingress class used in your application.
+
+[Traefik](https://traefik.io/traefik/) is used as the ingress class in HMCTS applications.
+
+The following traefik instances are available to use:
+
+| IngressClass | Default | Application configuration |
+|-|-|-|
+| traefik | Yes | Application will be reachable externally without VPN. End user will be required to authenticate to Azure AD before being redirected to the application |
+| traefik-private | No | Application will be available over VPN only |
+| traefik-no-proxy | No | Application will be reachable externally without VPN or authentication |
+
+### Protecting applications with Azure AD authentication
+
+If you do not set an ingressClass flag in code, it will default to traefik. 
+    
+To ensure end users must authenticate to Azure AD before they can view the application in their web browser, you must also set the `enableOAuth` flag to `true`
+
+```yaml
+nodejs:
+    ingressClass: traefik
+    disableTraefikTls: false
+    enableOAuth: true
+```
+
+The authentication with Azure AD is handled by an Azure AD App Registration in the HMCTS Demo Azure AD tenant. To allow end users to be authenticated to your application, you must add a redirect URI for your application to the Azure AD App Registration.
```

Comment:
the tenant should probably be linked (with a deep link containing either the tenant id or the tenant name so you don't get logged into the default tenant)